Main.py

```diff
@@ -209,10 +209,17 @@ async def addtime(ctx, time: str = None):
 
         day = _get_day(datestamp)
 
-        c.execute("INSERT OR REPLACE INTO Scores VALUES (?, ?, ?, ?)",
+        # Decide to REPLACE or INSERT based on if a score already exists at datestamp
+        has_record = c.execute(
+            "SELECT EXISTS (SELECT 1 FROM Scores WHERE ID = ? AND Date= ?)",
```

Comment:
this works (and yields a nice 1/0 response) but i would probably opt for something simpler:
```python
has_record = c.execute(
    "SELECT 1 FROM Scores WHERE ID = ? AND Date= ?",
    (member.id, _as_ymd(datestamp))
).fetchone()  # either (1,) or None
```
this will be a tuple (which is truth-y) if it exists, or None if it does not.